users/init.sls

```diff
@@ -145,7 +144,7 @@ users_{{ name }}_user:
     {% if 'homephone' in user %}
     - homephone: {{ user['homephone'] }}
     {% endif %}
-    {% if not user.get('createhome', True) %}
+    {% if not createhome -%}
     - createhome: False
     {% endif %}
```

Comment:
This conditional can be changed to a one-liner

```yaml
    - createhome: {{ createhome }}
```

Reply:
thanks for your input!
since this is my 1st real PR, PLMK if i'm doing something wrong :)